src/main/java/org/folio/services/domainevent/OldOrNewItem.java

```diff
@@ -0,0 +1,32 @@
+package org.folio.services.domainevent;
+
+import static io.vertx.core.json.Json.decodeValue;
+import static io.vertx.core.json.Json.encode;
+
+import org.folio.rest.jaxrs.model.HoldingsRecord;
+import org.folio.rest.jaxrs.model.Item;
+
+public class OldOrNewItem extends Item {
```

Comment:
The name `OldOrNewItem` seems a bit interesting. Not sure what a good alternative would be. `ItemWithInstanceId`, `DomainEventItem`, `ItemMessage`, is there any other better one? Also using inheritance is simpler here than composition, but do we have any concern about potential frequent `decodeValue` call in `fromItem` methods?

Reply:
Renamed to `ItemWithInstanceId`. Also, I'm using `@JsonUnwrapped` annotation now, so that composition is used now and decodeValue is not needed. Thanks.